collections/includes_value_test.ts

```diff
@@ -0,0 +1,44 @@
+import { includesValue } from "./includes_value.ts";
+import { assert, assertEquals } from "../testing/asserts.ts";
+
```

Comment:
I would feel more comfortable with some more test cases, maybe around non-primitive types, non-enumerable props etc without claiming that we did that everywhere else. The test file is just really short right now.